Fixed white header and menus

Internet-Anon opened the pull request.

tholinka commented:
I haven't been able to fix this myself, because I still have the old "header_beta" version myself. Can you readd the old tags, as it appears at least some people (me) are still getting the old version at this time?

Internet-Anon commented:
I have added the header_beta and some other stuff I had deleted. I tested it on the new version and it still works, see if this works for the old version as well.

tholinka commented on src/styles/header.scss:
```diff
@@ -11,6 +11,7 @@
 }
 
 #header,
+#header-beta,
```
header_beta, not header-beta

tholinka commented on src/styles/header.scss:
```diff
@@ -19,6 +20,7 @@
 
 	/* logo */
 	#header,
+	#header-beta .header-logo,
```
header_beta

tholinka replied:
Actually, not even sure we need this. As far as i can tell the `#logo_beta svg` takes care of any cases this would.

tholinka replied:
And I think I made a mistake when converting from `css` to `scss`, this should probably be just `.header-logo` anyway.

tholinka commented on src/styles/header.scss:
```diff
@@ -19,6 +20,7 @@
 
 	/* logo */
 	#header,
```
is this correct? this would get converted into `#header #header` when converting from `scss` to `css`. Should probably be `.erc-header__logo` I think?
